### DISCUSSION TITLE
Multi TRV setup not working as expected (TRVZB)

### DISCUSSION THREAD

**Discussion by a8ree** [jmcollin78/versatile_thermostat#673] [General]
I'm not sure if I have misconfigured my setup with the Sonoff TRVZB. I have multiple radiators in the same room, but as you see, despite the temperature on the VT being high, the thermostats are low
<img width="828" alt="Screenshot 2024-11-29 at 16 23 15" src="https://github.com/user-attachments/assets/98c2e7d2-059e-4b71-a8b4-21d90e89aed0">

I've tried increasing them but they are very unresponsive. About ten minutes after increasing, one of the TRVs opened - the rest remained closed.

I've configured these using the valve option and I've enabled the valve entities in HA, configuring on the open valve

<img width="577" alt="Screenshot 2024-11-29 at 16 23 37" src="https://github.com/user-attachments/assets/5146bf87-eae0-4420-8664-f95cd2650b1b">
<img width="510" alt="Screenshot 2024-11-29 at 16 23 57" src="https://github.com/user-attachments/assets/af811857-103d-4271-9003-57eaf2454bcc">
<img width="429" alt="Screenshot 2024-11-29 at 16 23 49" src="https://github.com/user-attachments/assets/0621ed8b-0b6d-4c19-8ea3-62db34c100a5">
<img width="492" alt="Screenshot 2024-11-29 at 16 23 43" src="https://github.com/user-attachments/assets/59ed5b5b-4566-4c0a-8e52-2e7b7c7fd6cb">

Can anyone see an issue with my config? I've tried to keep to defaults

**Comment by jmcollin78** [comment]
Hello @a8ree,

Sonoff TRVZB if an experimental feature in beta. The documenation is not written till now. Which beta do yo use ? 
You should configure the offset calibration entities of your TRV and eventually the close_degree entities. You have dedicated fields to enter them (in the same order than the underlyin and the opening_degree of course).

**Reply by a8ree** [reply]
Ah, thanks for pointing that out. I didn't appreciate that it was beta. I'll download the current beta and try it out
